targets/PythonSdk/templates/API.py.ejs

```diff
@@ -0,0 +1,34 @@
+import json
+import PlayFabErrors
+import PlayFabHTTP
+import PlayFabSettings
+
+<%- generateApiSummary("", api, "description")%>
+<% for(var i in api.calls) { var apiCall = api.calls[i];
+%><%- generateApiSummary("", apiCall, "summary")
+%><%- getDeprecationAttribute("", apiCall)
+%>def <%- apiCall.name %>(request, callback, customData = None, extraHeaders = None):
+<%- getRequestActions("    ", apiCall) %>
+    def wrappedCallback(playFabResult, error):
+<%- getResultActions("        ", apiCall, api) %>
+        callback(playFabResult, error)
```

Comment:
This template leaves an undesired breakline between
```
    def wrappedCallback(playFabResult, error):

        callback(playFabResult, error)
```

when getResultActions is empty.
(move the " %>" of getResultActions to the begining of the next line)